### PR TITLE
Adding self-hosting AT server capabilities

### PR DESCRIPTION
**Pull request by uriel1998**
Added environment variable for selfhosters of the AT server
Put in small checks to use environment variable instead of bsky.social if present 
Updated README, installation docs. 
Did NOT add to the installation script.

### CODE REVIEW

**Comment by bills-appworks** [comment]
Thank you for the pull request.
I will check the details separately, but first I would like to comment on the tab character.
Tab characters have been changed to spaces in some places, but the tab character plays an important role in this script. Therefore, changing this tab character to a space character is not acceptable.
https://github.com/uriel1998/bsky-sh-cli/blob/e0912b25baaf518cdb7c721c051684bd0ede37f3/lib/api/proxy.sh#L10
https://github.com/uriel1998/bsky-sh-cli/blob/e0912b25baaf518cdb7c721c051684bd0ede37f3/lib/bsky_core.sh#L10
https://github.com/uriel1998/bsky-sh-cli/blob/e0912b25baaf518cdb7c721c051684bd0ede37f3/lib/bsky_core.sh#L2720
https://github.com/uriel1998/bsky-sh-cli/blob/e0912b25baaf518cdb7c721c051684bd0ede37f3/lib/bsky_core.sh#L2804
https://github.com/uriel1998/bsky-sh-cli/blob/e0912b25baaf518cdb7c721c051684bd0ede37f3/lib/bsky_core.sh#L2878
https://github.com/uriel1998/bsky-sh-cli/blob/e0912b25baaf518cdb7c721c051684bd0ede37f3/lib/bsky_core.sh#L5410

File: README.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ Unless otherwise configured, it is assumed that the `bin` and `lib` directories
 
 Additionally, the tools listed in [Required tools](#required-tools) below must be executable.
 
-After installation, you can update to the latest version with the `bsky update` command. For details, please refer to the [installation documentation](INSTALL.md).
+After installation, you can update to the latest version with the `bsky update` command. For details, or if you are [self-hosting the AT server](https://rafaeleyng.github.io/self-hosting-a-bluesky-pds-and-using-your-domain-as-your-handle), please refer to the [installation documentation](INSTALL.md).
```
Comment (bills-appworks) [inline README.md:43]: This section explains how to update bsky-sh-cli itself, so I don't think it's necessary to add anything about self-hosting.

**Comment by bills-appworks** [comment]
If I approve your pull request, please note that for my release management purposes, I will merge it into a different branch (probably 0.13.0 or a feature branch) instead of the main branch.
After that, I will start preparing for the release.

**Comment by bills-appworks** [comment]
@uriel1998 
Do you have any comments on my comments so far?
If you don't respond, I'd like to add a feature based on your changes.
If I were to merge this request, I would like you to at least revert the tab character change, if possible.

**Comment by uriel1998** [comment]
Apologies; it's been a weird two weeks for me.  Will def. check it out, thanks for your patience.

**Comment by uriel1998** [comment]
Okay, should have the IFS tabs restored (stupid IDE auto-replaced them, that's why I didn't realize it).  Also added a check I missed where I'd hardcoded my AT endpoint, whoops.  

I apologize for any other errors I may have introduced; your skill level and sophistication with BASH *far* exceeds mine.

**Comment by bills-appworks** [comment]
Thank you for your kind response.
I will check it, make any necessary adjustments, and release it, so please wait.

File: README.md (hunk 1 above)
Comment (bills-appworks) [reply README.md:43]: I understand that this fix is ​​not a supplement to the update, but rather a supplement to installations in self-hosted environments that are separate from the update.

**Review by bills-appworks — COMMENTED** [review]
The release was announced:
https://bsky.app/profile/bsky-sh-cli.bills-appworks.net/post/3leg32uwekf22

**Pull request merged**